azure-kusto-data/azure/kusto/data/aio/client.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, kcsb: Union[KustoConnectionStringBuilder, str]):
         super().__init__(kcsb, True)
 
         self._session = ClientSession()
+        self._max_redirects = 0
```

Comment:
@AsafMah , is this a mistake?
Seems this should have been,
```suggestion
        self._session.max_redirects = 0
```

Reply:
This is not, in our async client you can't change the redirects at the session level, only at the request level.
Further down the file, we use _max_redirects for that.